src/App/Pages/Lock/LockFingerprintPage.cs

```diff
@@ -32,9 +33,23 @@ public LockFingerprintPage(bool checkFingerprintImmediately)
 
         public void Init()
         {
+            var image_helper = Helpers.OnPlatform(
+                        iOS: _deviceInfoService.HasFaceIdSupport ? "smile.png" : "fingerprint.png",
+                        Android: _deviceInfoService.HasFaceIdSupport ? "smile.png" : "fingerprint.png",
+                        Windows: "smile.png");
+            var text_helper = Helpers.OnPlatform(
+                        iOS: _deviceInfoService.HasFaceIdSupport ? AppResources.UseFaceIDToUnlock : AppResources.UseFingerprintToUnlock,
+                        Android: _deviceInfoService.HasFaceIdSupport ? AppResources.UseFaceIDToUnlock : AppResources.UseFingerprintToUnlock,
+                        Windows: "Unlock with Windows Hello");
+            var title_helper = Helpers.OnPlatform(
```

Comment:
This variable should use `camelCase` and also be named something more appropriate, such as `biometricTitle`.